sdk/core/loader/boot.cc

```diff
@@ -874,10 +1060,8 @@ extern "C" SchedulerEntryInfo loader_entry_point(const ImgHdr &imgHdr,
 	switcherKey.bounds()  = 1;
 	setSealingKey(imgHdr.scheduler(), Scheduler);
 	setSealingKey(imgHdr.allocator(), Allocator);
-	setSealingKey(imgHdr.allocator(),
-	              static_cast<SealingType>(0x1000000),
-	              0xff000000,
-	              sizeof(void *));
+	setSealingKey(
+	  imgHdr.allocator(), FirstDynamicSoftware, 0xff000000, sizeof(void *));
```

Comment:
`0xff000000` is `2**32 - FirstDynamicSoftware`?

Reply:
Yup

Reply:
Can it be spelled that way rather than as a "mysterious" hex constant?